src/content/1.5/code/agda/snippet18.agda

```diff
@@ -0,0 +1 @@
+m' (x , b) = (x , 42 , b)
```

Comment:
We do not have code in full.agda corresponding to this code.

Reply:
True, we also don't have code in `full.agda` corresponding to snippet06 because I imported `Agda.Builtin.Sigma` which defines `Σ` as follows:

```agda
record Σ {a b} (A : Set a) (B : A → Set b) : Set (a ⊔ b) where
  constructor _,_
  field
    fst : A
    snd : B fst
```
Maybe it would be better to rename `fst` and `snd` when importing `Σ` so we can include snippet06.  I'll try that.

Reply:
full.agda no longer imports `fst` and `snd` and now includes snippet06